### PR TITLE
Custom workflow d

### PR DESCRIPTION
**Pull request by soyacz**
PR for tracking tested workflows

See specific commits messages for details and results link

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@soyacz when I'll finish with #6719, we should use it here as well, and get it running on 2021.1.18

File: test-cases/longevity/custom_d1.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+gce_instance_type_db: 'n2-highmem-32'
+gce_instance_type_loader: 'e2-standard-4'
+
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_seed: '112'
+nemesis_interval: 5
+
+user_prefix: 'lukasz-3h-custom-d1'
+
+gce_n_local_ssd_disk_db: 6
```
Comment (fruch) [inline test-cases/longevity/custom_d1.yaml:20]: see: https://cloud.google.com/compute/docs/disks/local-ssd#:~:text=Each%20Local%20SSD%20disk%20is,that%20hosts%20your%20VM%20instance.

I don't think 6 would work

File: sdcm/utils/loader_utils.py [hunk 2]
```diff
@@ -175,7 +175,7 @@ def run_prepare_write_cmd(self):
         else:
             self._run_all_stress_cmds(write_queue, params={'stress_cmd': prepare_write_cmd,
                                                            'keyspace_num': keyspace_num,
-                                                           'round_robin': self.params.get('round_robin')})
+                                                           'round_robin': True})
```
Comment (fruch) [inline sdcm/utils/loader_utils.py:178]: do we still need this chnage

File: test-cases/longevity/custom_d1.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+user_prefix: 'lukasz-3h-custom-d1'
+
+gce_n_local_ssd_disk_db: 4
+gce_pd_ssd_disk_size_db: 1500
+gce_setup_hybrid_raid: true
+
+gce_image_db: 'https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/family/ubuntu-2004-lts'
+use_preinstalled_scylla: false
+scylla_d_overrides_files: [
+  'scylla-qa-internal/custom_d1/workload1/scylla.d/io_properties.yaml',
```
Comment (fruch) [inline test-cases/longevity/custom_d1.yaml:28]: we need to remove the io properties

File: vars/checkoutQaInternal.groovy [hunk 4]
```diff
@@ -4,6 +4,6 @@ def call(Map params = [:]) {
     dir("scylla-qa-internal") {
         git(url: 'git@github.com:scylladb/scylla-qa-internal.git',
             credentialsId:'b8a774da-0e46-4c91-9f74-09caebaea261',
-            branch: 'master')
+            branch: 'find-stress-command')
```
Comment (fruch) [inline vars/checkoutQaInternal.groovy:7]: you still have things needed to be merge into `scylla-qa-internal` ? please open PRs for them

File: test-cases/longevity/custom_d1.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+stress_cmd: "cassandra-stress user profile=scylla-qa-internal/custom_d1/posts_dataset.yaml cl=QUORUM 'ops(insert=5,update1=3,read1=50,delete1=1)' duration=720m -rate 'threads=100' -pop 'dist=gauss(0..24M)'"
+gce_instance_type_db: 'n2-highmem-32'
+gce_instance_type_loader: 'e2-standard-4'
+
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_seed: '112'
+nemesis_interval: 5
+nemesis_during_prepare: false
+
+user_prefix: 'lukasz-3h-custom-d1'
```
Comment (fruch) [inline test-cases/longevity/custom_d1.yaml:19]: you can call it `hybrib-3h-custom-d1`

**Comment by fruch** [comment]
@soyacz let's align what's needed in this PR, and get it merged.

File: sdcm/utils/loader_utils.py (hunk 2 above)
Comment (soyacz) [reply sdcm/utils/loader_utils.py:178]: I adapted stress_cmd so we don't need it

**Comment by soyacz** [comment]
@fruch if we merge it I'll test https://github.com/scylladb/scylla-cluster-tests/pull/6752 if fits it's purpose, as currently fighting with it is problematic and takes too much time.

**Review by fruch — APPROVED** [review]
Lgtm

**Pull request merged**